modules/core/core-api/src/main/java/com/enonic/xp/xml/parser/XmlObjectParser.java

```diff
@@ -29,10 +33,17 @@
 {
     private static final SchemaValidator VALIDATOR = new SchemaValidator();
 
+    private final String namespace;
+
     private String systemId;
 
     private CharSource source;
 
+    public XmlObjectParser( final String namespace )
```

Comment:
add default ctor that, if used, does not force any namespace. `namespace = null`